src/styles/main.scss

```diff
@@ -0,0 +1,58 @@
+@import "./vars/vars";
+@import "./reset/reset";
+@import "./mixins/mixins";
+
+@keyframes clockRotate {
+  from {
+    transform: rotate(0);
+  }
+
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+.clock {
+  height: 400px;
+  width: 400px;
```

Comment:
Add variables for these values.
Also, it would be great to add variables for all important values, such as colors, animation time etc.